src/databricks/labs/ucx/contexts/application.py

```diff
@@ -250,6 +251,19 @@ def tables_migrator(self):
             self.principal_acl,
         )
 
+    @cached_property
+    def acl_migrator(self):
+        return ACLMigrator(
```

Comment:
can you integrate it with existing tables migrate process? unit tests should pass. add more if we need to increase coverage.

Reply:
I created it as a separate one because there are some nuances (for hms_fed). I can work to combine the common code.